zerver/lib/notifications.py

```diff
@@ -299,11 +299,11 @@ def do_send_missedmessage_events_reply_in_zulip(user_profile: UserProfile,
     context = common_context(user_profile)
     context.update({
         'name': user_profile.full_name,
-        'messages': build_message_list(user_profile, missed_messages),
```

Comment:
I'd make this move in a separate commit. (I assume context.messages is now being set somewhere else?)
Though I also suspect we don't need this change, and should do the appropriate `if` statements in the templates.